tasks/core/chroot_clean/chroot_clean.sh

```diff
@@ -10,3 +10,6 @@ rm "${DS_WORK}/rootfs/${QEMU_STATIC_PATH}"
 rm -rf ${DS_WORK}/rootfs/dev/*
 rm -rf ${DS_WORK}/rootfs/tmp/*
 rm -rf ${DS_WORK}/rootfs/run/*
+
+# Remove leftover apt lists
+rm -rf ${DS_WORK}/rootfs/var/lib/apt/lists/*
```

Comment:
Are there any other existing files that we can/should clean up? For example, this is what Image Replicator does to clean basically all of the caches, logs, etc. I'm not sure how many of these would be needed for a bootstrapping like this.